### PR TITLE
Add eye color option for people with heterochromia

### PR DESCRIPTION
**Pull request by scpketer**

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -11929,11 +11936,20 @@ label monika_player_appearance:
...
                     m 3hub "Oh! That's a beautiful color, [player]!"
                     m 2eub "I'm sure I could get lost for hours, staring into your [persistent._mas_pm_eye_color] eyes."
                     m 3hua "Now, onto my next question--"
 
+                "I have heterochromia.":
+                    $ persistent._mas_pm_has_heterochromia = True
+
+                    m 1sub "Really? {w=0.5}{nw}"
+                    extend 3hua "You're so special, [player]!"
+                    m 5rtc "If I recall correctly, less than one percent of people in world have this curious feature..."
```
Comment (Rai99) [inline Monika After Story/game/script-topics.rpy:11950]: Dissolve into body pose 5. 
```suggestion
                    m 5rtc "If I recall correctly, less than one percent of people in the world have this curious feature..."
```

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -11929,11 +11936,20 @@ label monika_player_appearance:
...
+                    m 1hua "...So you must be really rare, ehehe~"
```
Comment (Rai99) [inline Monika After Story/game/script-topics.rpy:11951]: Dissolve out of body pose 5.

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -11929,11 +11936,22 @@ label monika_player_appearance:
...
                 "My eyes are another color.":
                     $ persistent._mas_pm_eye_color = ask_color("What color are your eyes?")
+                    $ persistent._mas_pm_has_heterochromia = False
 
                     m 3hub "Oh! That's a beautiful color, [player]!"
                     m 2eub "I'm sure I could get lost for hours, staring into your [persistent._mas_pm_eye_color] eyes."
                     m 3hua "Now, onto my next question--"
 
+                "I have heterochromia.":
+                    $ persistent._mas_pm_has_heterochromia = True
```
Comment (Booplicate) [inline Monika After Story/game/script-topics.rpy:11946]: I'm not sure how useful this whole thing is, but if we do it, can as well ask eyes colours.

Comment (scpketer) [reply Monika After Story/game/script-topics.rpy:11946]: Right, I think we can.

**Comment by scpketer** [comment]
Regarding 560b2d6: I'm working on two eye color choice dialog, however, I'd really appreciate some assistance with dynamically generated menus. Besides, there still an unresolved question on how should we display the user-defined color menu item. I don't actually see 'I have another eye color.../...and another eye color.' really fitting in, and that's what worries me.

**Comment by Booplicate** [comment]
> There is another potential pitfall - what if the player has just one eye?

Please, don't overcomplicate this. These change are already for a niche case. Just ask colour of the right eye, left eye and save the tuple.

**Comment by scpketer** [comment]
You're right, I already decided not to add anything like that, too.

**Review by multimokia — APPROVED** [review]

**Review by jmwall24 — APPROVED** [review]

**Pull request merged**